cross-account-region-data-pipeline/utils/application_config.py

```diff
@@ -0,0 +1,40 @@
+from typing import Any, Dict
+import json
+
+class GetApplicationParameters():
```

Comment:
Similar to [this](https://github.com/aws-samples/aws-ddk-examples/pull/9) - could DDK Config class be used here?